packages/tailwindcss-language-syntax/tests/__snapshots__/syntax.test.ts.snap

```diff
@@ -833,46 +833,130 @@ exports[`@theme 1`] = `
-^^^^                                   1: source.css.tailwind
+^^^                                    2: source.css.tailwind meta.at-rule.theme.body.tailwind meta.at-rule.keyframes.body.css
+  ^                                    1: punctuation.section.keyframes.end.bracket.curly.css
+
+}
+^                                      1: source.css.tailwind meta.at-rule.theme.body.tailwind punctuation.section.theme.end.bracket.curly.tailwind
+
+
+^                                      1: source.css.tailwind
+
+@theme {
+^^^^^^^^                               4: source.css.tailwind
+^^^^^^                                 2: keyword.control.at-rule.theme.tailwind
+^                                      1: punctuation.definition.keyword.css
+       ^                               1: meta.at-rule.theme.body.tailwind punctuation.section.theme.begin.bracket.curly.tailwind
+
+  /** Comment 0 */
+^^^^^^^^^^^^^^^^^^                     4: source.css.tailwind meta.at-rule.theme.body.tailwind
+  ^^^^^^^^^^^^^^^^                     3: comment.block.css
+  ^^                                   1: punctuation.definition.comment.begin.css
+                ^^                     1: punctuation.definition.comment.end.css
+
+
+^                                      1: source.css.tailwind meta.at-rule.theme.body.tailwind
+
+  /** Comment 1 */
+^^^^^^^^^^^^^^^^^^                     4: source.css.tailwind meta.at-rule.theme.body.tailwind
+  ^^^^^^^^^^^^^^^^                     3: comment.block.css
+  ^^                                   1: punctuation.definition.comment.begin.css
+                ^^                     1: punctuation.definition.comment.end.css
+
+  --color-1: red;
+^^^^^^^^^^^^^^^^^                      6: source.css.tailwind meta.at-rule.theme.body.tailwind
+  ^^^^^^^^^                            1: variable.css
+           ^                           1: punctuation.separator.key-value.css
+             ^^^                       1: meta.property-value.css support.constant.color.w3c-standard-color-name.css
+                ^                      1: punctuation.terminator.rule.css
+
+
+^                                      1: source.css.tailwind meta.at-rule.theme.body.tailwind
+
+  /** Comment 2 */
+^^^^^^^^^^^^^^^^^^                     4: source.css.tailwind meta.at-rule.theme.body.tailwind
+  ^^^^^^^^^^^^^^^^                     3: comment.block.css
+  ^^                                   1: punctuation.definition.comment.begin.css
+                ^^                     1: punctuation.definition.comment.end.css
+
+  --color-2: green;
+^^^^^^^^^^^^^^^^^^^                    6: source.css.tailwind meta.at-rule.theme.body.tailwind
+  ^^^^^^^^^                            1: variable.css
+           ^                           1: punctuation.separator.key-value.css
+             ^^^^^                     1: meta.property-value.css support.constant.color.w3c-standard-color-name.css
+                  ^                    1: punctuation.terminator.rule.css
```

Comment:
This feels wrong? Shouldn't that point to the `.`?

Reply:
Oh wait I'm a dumb dumb, thanks GitHub diff viewer. It's pointing to the `;` of course.

This is what I see due to word-wrapping lol 
<img width="802" alt="image" src="https://github.com/user-attachments/assets/d78337f7-68cd-4cf8-96ab-3763a12638ee" />